### DISCUSSION TITLE
distracting fade to black on new app version

### DISCUSSION THREAD

**Discussion by Squ1dw4rd99** [giejay/Immich-Android-TV#63] [Q&A]
Hello, 
I just upgraded from 1.35 to 1.38 and noticed at the bottom of every image there is a fade to black now (as well as the current picture number), and that cannot be turned off in the settings..
Is it possible to create a toggle to turn off this new "feature"? I want to view the images how I took them and not with a dark fade at the bottom.

edit: I noticed there is a view option called "Show media count". I have that turned off, but it still shows the count, so I guess that is the problem here

**Comment by giejay** [comment]
What do you mean faded background? Can you make a photo?

**Reply by Squ1dw4rd99** [reply]
it looks like this (the original image shows a uniform white sky, here you can see the fade, ignore the pixel patterns those are not visible irl)!

**Comment by giejay** [comment]
Hmm not sure what could cause this. Can you turn of the option "slide into image"?

**Comment by Squ1dw4rd99** [comment]
> Hmm not sure what could cause this. Can you turn of the option "slide into image"?

As far as I can see, this option is only available under the screensaver options and when I toggle it there is no impact on the normal image viewing (the images still slide and the dark fade is still there)

**Comment by Squ1dw4rd99** [comment]
I noticed another thing: the option "show media count" in normal viewing doesn't work anymore, but when you disable it in the screensaver settings, it gets disabled for normal viewing... maybe something got mixed up with the settings in the update

**Comment by giejay** [comment]
Found both the issues. It was because of a PR enhancements in the screensaver (showing a clock etc.). That added a gradient overlay to all photos. For now, I just enabled the overlay in screensaver mode, not in regular photo mode. Also the other issue with media count/title is fixed in 1.39
